docs/.vitepress/vitepress/components/theme-editor/ep-theme-color-box.vue

```diff
@@ -0,0 +1,66 @@
+<script lang="ts" setup>
+import { computed, defineProps } from 'vue'
+import { TinyColor } from '@ctrl/tinycolor'
+import { useClipboard } from '@vueuse/core'
+import { ElMessage } from 'element-plus'
+import type { CSSProperties } from 'vue'
+
+import type { HSV } from '@ctrl/tinycolor'
+
+const props = defineProps<{
+  // hsb
+  hsv?: HSV
+}>()
+
+const hex = computed(() => {
+  return new TinyColor(props.hsv).toHexString()
+})
+
+const styles = computed(() => {
+  return {
+    '--uc-c-bg': hex.value,
+  } as CSSProperties
+})
+
+const { copy } = useClipboard()
+
+const copyHex = () => {
+  const text = hex.value?.toUpperCase() || ''
+  copy(text)
+
+  ElMessage.success({
+    dangerouslyUseHTMLString: true,
+    message: `<div class="flex justify-center items-center">Copy ${text}
```

Comment:
Can we use VNode / JSX here?

Reply:
I don't think JSX can be supported here, and I think HTML is more readable.